### PR TITLE
Subsquid

### PR DESCRIPTION
**Pull request by Sociopacific**

### CODE REVIEW

File: public/env.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
-  "BLOCKCHAIN_URL": "wss://ws.framenode-2.r0.dev.sora2.soramitsu.co.jp",
-  "SUBQUERY_ENDPOINT": "https://api.subquery.network/sq/sora-xor/sora-dev",
+  "BLOCKCHAIN_URL": "wss://mof2.sora.org",
```
Comment (stefashkaa) [inline public/env.json:2]: BLOCKCHAIN_URL - please revert this line

File: src/services/indexer/subsquid/client.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export type { Client, OperationResult, TypedDocumentNode, AnyVariables } from '@urql/core';
+
+const createSubscriptionClient = (url: string) => {
+  const wsUrl = url.replace(/^http/, 'ws');
+
+  return createWSClient({
+    url: wsUrl,
+    lazy: true,
+    retryAttempts: 25,
```
Comment (stefashkaa) [inline src/services/indexer/subsquid/client.ts:14]: do we have the same configuration for subsquid? maybe it's better to move it somewhere in common place? what do you think?

**Review by stefashkaa — APPROVED** [review]

File: public/env.json (hunk 1 above)
Comment (Sociopacific) [reply public/env.json]: Reverted

File: src/services/indexer/subsquid/client.ts (hunk 2 above)
Comment (Sociopacific) [reply src/services/indexer/subsquid/client.ts]: Moved to consts/indexer.ts

**Pull request merged**